src/components/modules/contributionsBanners/AusMomentContributionsBanner.tsx

```diff
@@ -432,14 +433,18 @@ const socialShare = (
     </div>
 );
 
-const support = (
+const urlWithTracking = (baseUrl: string, tracking: BannerTracking): string => {
+    return `${baseUrl}?acquisitionData=%7B%22source%22%3A%22${tracking.platformId}%22%2C%22componentType%22%3A%22ACQUISITIONS_ENGAGEMENT_BANNER%22%2C%22componentId%22%3A%22${tracking.ophanComponentId}%22%2C%22campaignCode%22%3A%22${tracking.campaignCode}%22%7D&INTCMP=${tracking.campaignCode}}`;
```

Comment:
actually, confusingly `ophanComponentId` will be the component type, i.e. `22ACQUISITIONS_ENGAGEMENT_BANNER`.
Please can you use `campaignCode` for the `componentId`. This has the form `testname_variantname`